src/kernel.h

```diff
@@ -24,6 +24,8 @@ bool IsProtocolV03(unsigned int nTimeCoinStake);
 bool IsProtocolV04(unsigned int nTimeBlock);
 // Whether a given transaction is subject to new v0.5 protocol
 bool IsProtocolV05(unsigned int nTimeTx);
+// Whether a given block or transaction is subject to new v0.6 protocol
+bool IsProtocolV06(unsigned int nTimeTx);
```

Comment:
I didn't notice the difference between nTimeBlock and nTimeTx.

Why would you ever use nTimeTx?
It is not reliable as the sender can pick any time he likes.